tests/unit/test_error_handling.py

```diff
@@ -23,5 +36,165 @@ def test_empty_model():
 
 def test_unknown_component_types():
     model = PowerGridModel(input_data={})
-    with pytest.raises(KeyError, match=r"artificial_type") as e:
-        model.calculate_power_flow(output_component_types={"artificial_type"})
+    with pytest.raises(KeyError, match=r"fake_type"):
+        model.calculate_power_flow(output_component_types={"fake_type"})
+
+
+def test_handle_conflict_voltage_error():
+    node_input = initialize_array("input", "node", 2)
+    node_input["id"] = [0, 1]
+    node_input["u_rated"] = [10.0e3, 20.0e3]
+
+    line_input = initialize_array("input", "line", 1)
+    line_input["id"] = [2]
+    line_input["from_node"] = [0]
+    line_input["to_node"] = [1]
+
+    with pytest.raises(ConflictVoltage):
+        PowerGridModel(input_data={"node": node_input, "line": line_input})
+
+
+@pytest.mark.skip(reason="TODO")
+def test_handle_missing_case_for_enum_error():
+    pass
```

Comment:
it is not easy to find an example that will work here - now and in the future. what should we do with these tests?

Reply:
I'd say "Create corresponding issue/tickets and put it on higher priority than before"

Reply:
The thing is that i don't know whether we can actually find these cases.. best we can do to test the regex is hack such that it is raised and then mock the PGM response i guess? Or do you have a better idea?